ASP.NET API/WAVC_WebApi/Controllers/FriendsManagerController.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WAVC_WebApi.Data;
+using WAVC_WebApi.Models;
+
+namespace WAVC_WebApi.Controllers
+{
+    [Route("api/[controller]")]
+    [Authorize]
+    [ApiController]
+    public class FriendsManagerController : ControllerBase
```

Comment:
I would rename controller just to FriendsController
calling  GET /api/FriendManager/Friends looks weird